controllers/kogitoinfra_controller.go

```diff
@@ -52,6 +47,10 @@ type KogitoInfraReconciler struct {
 // +kubebuilder:rbac:groups=eventing.knative.dev,resources=triggers,verbs=get;list;watch;create;delete;update
 // +kubebuilder:rbac:groups=sources.knative.dev,resources=sinkbindings,verbs=get;list;watch;create;delete;update
 // +kubebuilder:rbac:groups=integreatly.org,resources=grafanadashboards,verbs=get;create;list;watch;create;delete;update
+// +kubebuilder:rbac:groups=mongodb.com,resources=mongodbs,verbs=get;create;list;watch;delete
```

Comment:
```suggestion
// +kubebuilder:rbac:groups=mongodb.com,resources=mongodb,verbs=get;create;list;watch;delete
```

Reply:
See the explaination: https://kie.zulipchat.com/#narrow/stream/232676-kogito/topic/operator.20and.20mongo

Reply:
Yeah, I can fix this, but not sure about the fetch since they enforced singular for plurals. The k8s API is expecting plurals when fetching. I'll see what I can do about it.